app/src/main/res/values/strings.xml

```diff
@@ -627,7 +627,12 @@ Upload your first media by tapping on the add button.</string>
   <string name="pan_and_zoom_to_adjust">Pan and zoom to adjust</string>
   <string name="exit_location_picker">Exit location picker</string>
   <string name="select_location_location_picker">Select location</string>
+  <string name="show_in_map_app">SHOW IN MAP APP</string>
```

Comment:
-> Show in map app
Uppercasing should be styled by the app, rather than hard-coded in strings.xml
I believe you can take inspiration similar examples in the app's code.